README.md

```diff
@@ -1,2 +1,53 @@
 # TesasSemble
 assemble in the 4th dimension 
+
+
+### Environment
+
+To install the conda environment, do
+
+```
+conda env create -f resources/environment.yml
```

Comment:
This should have an explicit name so that you can keep track of the env. For example :
```
 conda env create -n tesas --file resources/environment.yml
```

Reply:
yeah you're right, I did this on my own but forgot to write that flag in the readme